CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusSteppables.py

```diff
@@ -354,10 +358,11 @@ def step(self, mcs):
             # Test for infection secretion
             if cell.dict['Assembled'] > cell_infection_threshold:
                 cell.type = self.INFECTEDSECRETING
+                cell.dict['ck_production'] = max_ck_secrete_infect
```

Comment:
Redundant, unless planning to incorporate into viral secretion. If so, please mark accordingly.

Reply:
Commented out in case we want to add it to viral secretion later